### PR TITLE
follow up #17276

### PR DESCRIPTION
**Pull request by ringabout**

### CODE REVIEW

File: lib/std/private/jsutils.nim [hunk 1]
```diff
@@ -40,4 +40,12 @@ when defined(js):
 
   proc isInteger*[T](x: T): bool {.importjs: "Number.isInteger(#)".}
 
-  proc isSafeInteger*[T](x: T): bool {.importjs: "Number.isSafeInteger(#)".}
+  proc isSafeInteger*[T](x: T): bool {.importjs: "Number.isSafeInteger(#)".} =
+    runnableExamples:
+      import std/jsffi
+      assert not "123".toJs.isSafeInteger
```
Comment (ghost) [inline lib/std/private/jsutils.nim:46]: I might be wrong, but isn't `doAssert` is preferred to `assert` in `runnableExamples`?

Comment (ringabout) [reply lib/std/private/jsutils.nim:46]: docs changed,

Comment (ringabout) [reply lib/std/private/jsutils.nim:46]:
```nim
runnableExamples:
  doAssert foo() # bad
  assert foo() # preferred

runnableExamples("-d:danger"):
  doAssert foo() # `assert` would be disabled here, so `doAssert` makes more sense
```

Ref https://nim-lang.github.io/Nim/contributing.html

Comment (konsumlamm) [reply lib/std/private/jsutils.nim:46]: See https://github.com/nim-lang/Nim/pull/17077.

File: lib/std/private/jsutils.nim [hunk 2]
```diff
@@ -40,4 +40,12 @@ when defined(js):
...
+      assert 123.toJs.isSafeInteger
+      assert 9007199254740991.toJs.isSafeInteger
+      assert not 9007199254740992.toJs.isSafeInteger
+
+  let maxSafeInteger* {.importc: "Number.MAX_SAFE_INTEGER".} : int64
```
Comment (ringabout) [inline lib/std/private/jsutils.nim]:
```suggestion
  let maxSafeInteger* {.importjs: "Number.MAX_SAFE_INTEGER".} : int64
```

**Pull request merged**